models/articles.model.js

```diff
@@ -32,3 +32,11 @@ exports.updateArticleById = (article_id, increase_votes) => {
     return article.rows[0];
   });
 };
+
+exports.fetchAllArticles = () => {
+  let queryStr =
+    " SELECT articles.*, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id=comments.article_id GROUP BY articles.article_id ORDER BY created_at DESC;";
```

Comment:
we don't want to return the body with the articles in this case so need to select manually